OpenRA.Mods.Common/Activities/Turn.cs

```diff
@@ -19,21 +19,26 @@ public class Turn : Activity
 	{
 		readonly IDisabledTrait disablable;
 		readonly IFacing facing;
+		readonly Mobile mobile;
```

Comment:
`Turn` is only used by `Mobile`, which is a `PausableConditionalTrait<T>`, and `Aircraft`, which is neither.
Since we're now forced to reference `Mobile` directly, we might as well remove `disablable` which doesn't cover any further cases in practice.